homeassistant/components/hassio/__init__.py

```diff
@@ -323,6 +336,52 @@ async def async_create_backup(
     return await hassio.send_command(command, payload=payload, timeout=None)
 
 
+@bind_hass
```

Comment:
Consider not adding `bind_hass`. It's legacy now that we want to import all names as normal instead of accessing `hass.components`.

Reply:
This is there for consistency with all other functions defined here.
I have plans on doing major refacor/cleanup to the entire hassio integration, and changing that will be a part of that effort.